paasta_tools/tron_tools.py

```diff
@@ -256,7 +257,16 @@ def get_schedule(self):
         return self.config_dict.get('schedule')
 
     def get_monitoring(self):
-        return self.config_dict.get('monitoring')
+        monitoring_conf = self.config_dict.get('monitoring', {})
+        if not monitoring_conf:
+            monitoring_conf = monitoring_tools.read_monitoring_config(
+                self.service,
+                soa_dir=self.soa_dir,
+            )
+            # PAASTA-15158: For now, we do not expose more than the 'team' from
```

Comment:
I don't think this is worth doing and I think it will cause more confusion than you think.

This means that we'll get a team, and start doing stuff, but we *won't* be using any of the other settings in monitoring.yaml, so emails will go to a different place than they have configured, etc.

Reply:
Alright, I'll switch monitoring on for all tron jobs.